samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineCustomJobSample.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aiplatform;
+
+// [START aiplatform_create_training_pipeline_custom_job_sample]
+import com.google.cloud.aiplatform.v1beta1.LocationName;
+import com.google.cloud.aiplatform.v1beta1.Model;
+import com.google.cloud.aiplatform.v1beta1.ModelContainerSpec;
+import com.google.cloud.aiplatform.v1beta1.PipelineServiceClient;
+import com.google.cloud.aiplatform.v1beta1.PipelineServiceSettings;
+import com.google.cloud.aiplatform.v1beta1.TrainingPipeline;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+public class CreateTrainingPipelineCustomJobSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "PROJECT";
+    String displayName = "DISPLAY_NAME";
+    String modelDisplayName = "MODEL_DISPLAY_NAME";
+    String containerImageUri = "CONTAINER_IMAGE_URI";
+    String baseOutputDirectoryPrefix = "BASE_OUTPUT_DIRECTORY_PREFIX";
+    createTrainingPipelineCustomJobSample(
+        project, displayName, modelDisplayName, containerImageUri, baseOutputDirectoryPrefix);
+  }
+
+  static void createTrainingPipelineCustomJobSample(
+      String project,
+      String displayName,
+      String modelDisplayName,
+      String containerImageUri,
+      String baseOutputDirectoryPrefix)
+      throws IOException {
+    PipelineServiceSettings settings =
+        PipelineServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
```

Comment:
Why are we doing this?

Reply:
Acknowledged.

AI Platform doesn't have a global hostname, only locale-specific hostnames.

Reply:
Consider adding a comment to this effect.

Reply:
Acknowledged.

We've used this technique on every single sample in this repo. I'll create a GitHub issue to go back and add a comment to all of the places where we do this.